joss_paper/paper.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'SunPy: A Python package for Solar Physics'
+tags:
+  - Python
+  - Astronomy
+  - Solar physics
+authors:
+  - name: Stuart J. Mumford
+    orcid: 0000-0003-4217-4642
+    affiliation: "1, 2"
+  - name: Nabil Freij
+  - name: Steven Christe
+  - name: Jack Ireland
+  - name: Florian Mayer
+  - name: Keith Hughitt
+  - name: Albert Y. Shih
+  - name: Daniel Ryan
+  - name: Simon Liedtke
+  - name: David Pérez-Suárez
+  - name: Pritish Chakraborty
+  - name: Vishnunarayan K I.
+  - name: Andrew Inglis
+  - name: Punyaslok Pattnaik
+  - name: Brigitta Sipocz
+  - name: Rishabh Sharma
+  - name: Andrew Leonard
+  - name: David Stansby
+  - name: Russell Hewett
+  - name: Alex Hamilton
+  - name: Laura Hayes
+  - name: Asish Panda
+  - name: Matt Earnshaw
+  - name: Nitin Choudhary
+  - name: Ankit Kumar
+  - name: Prateek Chanda
+  - name: Md Akramul Haque
+  - name: Michael S Kirk
+  - name: Michael Mueller
+  - name: Sudarshan Konge
+  - name: Rajul Srivastava
+  - name: Yash Jain
+  - name: Samuel Bennett
+  - name: Ankit Baruah
+  - name: Will Barnes
+  - name: Michael Charlton
+  - name: Cubostar
+  - name: Shane Maloney
+  - name: Nicky Chorley
+  - name: Himanshu
+  - name: Sanskar Modi
+  - name: James Paul Mason
+  - name: Naman9639
+  - name: Yash
+  - name: Jose Ivan Campos Rozo
```

Comment:
Name: Jose Ivan Campos Rozo
orcid: 0000-0001-8883-6790
affiliation: Institut für Physik/IGAM - Karl-Franzens University of Graz, Austria